### DISCUSSION TITLE
Embedding images in gallery view from obsidian vault

### DISCUSSION THREAD

**Discussion by TommyTommy1234** [marcusolsson/obsidian-projects#610] [Help] [answered]
Hello, loving the plugin! I was wondering, is there a way to embed images from the obsidian vault in the gallery view? Right now I embed images from my dropbox but a drag n drop functionality or being able to use obsidian links would be so great!

Thanks,
Joel

**Comment by marcusolsson** [comment, marked as answer]
If you set the cover to be an internal link, it should work. Let me know if not! 

For example:

```yaml
cover: "[[profile.png]]"
```

**Reply by Acylation** [reply]
More formats will be supported in the next release, by PR #822 

- Wiki link: [[internal link|text]]
- Markdown link: [text](internal link)
- External link (web link): [text](https://...) and [text](http://...)
- Plain web link: https://... & http://...
